Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabLinks.cshtml

```diff
@@ -32,8 +32,8 @@
             @foreach (var l in Model.LinkedEstablishments)
             {
                 <tr class="govuk-table__row">
-                    <td data-label="URN" class="govuk-table__cell">@Html.ActionLink(l.Urn.ToString(), "Details", new {id = l.Urn})</td>
-                    <td data-label="Establishment name" class="govuk-table__cell">@Html.ActionLink((l.EstablishmentName ?? "-"), "Details", new {id = l.Urn})</td>
+                    <td data-label="URN" id=@l.Urn  class="govuk-table__cell">@Html.ActionLink(l.Urn.ToString(), "Details", new {id = l.Urn})</td>
```

Comment:
Does this `id` value need quotes around the value?

Reply:
when I didn't use the quotes it was working as expected. I tried using quotes the results are same. Happy to change it either ways

Reply:
The @ symbol means we are going into C#/Razor code at this point , the id will be set to the Urn of l